### ISSUE TITLE
[4.13.1 (90346)] Unable pull images from docker-registry

### ISSUE THREAD

**Issue by budarin** [docker/for-win#13053]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 374513F7-8009-484F-8C36-46187B87809E/20221106122756

### Actual behavior

Push an image to the docker registry v2

```
docker push localhost:4000/raketa/service:1.31.16;
docker push localhost:4000/raketa/service:latest;
```

![image](https://user-images.githubusercontent.com/8055157/200170967-95e4e671-cfd0-41a4-8c66-400bb00b507f.png)

then try to pull it

```
$ docker pull localhost:4000/raketa/service
Using default tag: latest
Error response from daemon: unknown image in /v1.41/images/create?fromImage=localhost%3A4000%2Fraketa%2Fservice&tag=latest
```

### Expected behavior

images must be pulled with `docker pull` command as it was before the last update

### Information
  - Windows Version: Windows 10 Pro 21H2 19044.2130
  - Docker Desktop Version: 4.13.1 (90346)
  - WSL2 or Hyper-V backend?  WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: Windows

### Output of `& "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check`

```
[2022-11-06T12:40:14.531164700Z][com.docker.diagnose.exe][I] set path configuration to OnHost
Starting diagnostics

[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0002: does the bootloader have virtualization enabled?
[SKIP] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0022: is the Virtual Machine Platform Windows Feature enabled?
[PASS] DD0021: is the WSL 2 Windows Feature enabled?
[PASS] DD0024: is WSL installed?
[PASS] DD0025: are WSL distros installed?
[PASS] DD0026: is the WSL LxssManager service running?
[PASS] DD0029: is the WSL 2 Linux filesystem corrupt?
[PASS] DD0035: is the VM time synchronized?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0013: is the $PATH ok?
[PASS] DD0003: is the Docker CLI working?
[PASS] DD0005: is the user in the docker-users group?
[FAIL] DD0014: are the backend processes running? 1 error occurred:
        * vpnkit.exe is not running


[PASS] DD0007: is the backend responding?
[PASS] DD0008: is the native API responding?
[PASS] DD0009: is the vpnkit API responding?
[PASS] DD0010: is the Docker API proxy responding?
[PASS] DD0006: is the Docker Desktop Service responding?
[PASS] DD0012: is the VM networking working?
[SKIP] DD0030: is the image access management authorized?
[PASS] DD0033: does the host have Internet access?
[PASS] DD0002: does the bootloader have virtualization enabled?
[PASS] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0022: is the Virtual Machine Platform Windows Feature enabled?
[PASS] DD0021: is the WSL 2 Windows Feature enabled?
[PASS] DD0024: is WSL installed?
[PASS] DD0025: are WSL distros installed?
[PASS] DD0026: is the WSL LxssManager service running?
[PASS] DD0029: is the WSL 2 Linux filesystem corrupt?
[PASS] DD0035: is the VM time synchronized?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0032: do Docker networks overlap with host IPs?

Please investigate the following 1 issue:

1 : The test: are the backend processes running?
    Failed with: 1 error occurred:
        * vpnkit.exe is not running



Not all of the backend processes are running.
```

**Comment by budarin** [comment]
https://forums.docker.com/t/docker-registry-error-response-from-daemon-unknown-image-in-v1-41-images-create-fromimage/130944

**Comment by nicks** [comment]
thanks for the report! I can reproduce this. Agreed that it's likely the same root cause as https://github.com/docker/for-mac/issues/6540 but want to confirm.

**Comment by developinua** [comment]
The same issue for me.
I am running the registry locally on the macOS Ventura M1 chip.
I can tag and push to my local registry but I can't pull that image from it.
The image exists in my local private docker registry file directory.

**Comment by maxcleme** [comment]
Closing this issue because a fix has been released in Docker Desktop 4.15.0. See [release notes](https://docs.docker.com/desktop/release-notes/) for more details.
